### ISSUE TITLE
Debugger Fails Cryptically:  Error: Compilation shimmedcompilation has unreliable source order

### ISSUE THREAD

**Issue by zach-is-my-name** [trufflesuite/truffle#2879]
- [ X] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

> What the issue is, in broad strokes.
Debugger Fails Cryptically:  Error: Compilation shimmedcompilation has unreliable source order
## Steps to Reproduce
![Screenshot from 2020-03-11 15-30-36](https://user-images.githubusercontent.com/16063092/76456491-70babe00-63ad-11ea-97bf-af94946c1549.png)

## Expected Behavior

> What you expected to happen.
Debug the transaction
## Actual Results
Screenshot ^
What actually happened. Please give examples and support it with screenshots, copied output or error messages.

## Environment

**ubuntu 16.04
Ganache CLI v6.5.0 (ganache-core: 2.6.0)
Truffle v5.1.16 (core: 5.1.16)
Solidity v0.5.16 (solc-js)
Node v12.6.0
Web3.js v1.2.1**

**Comment by haltman-at** [comment]
Thanks for the report!  This definitely shouldn't happen.  Fortunately I've already managed to reproduce the problem, so hopefully I can figure out how this slipped through...

**Comment by haltman-at** [comment]
OK, just put up #2880 to fix this, so this should be fixed and out quickly!

**Comment by JackBekket** [comment]
Hi, I've experiencing same error when try to debug tx

Truffle v5.1.45 (core: 5.1.45)
Ganache v2.5.4

**Comment by haltman-at** [comment]
@JackBekket Does this issue still exist on current Truffle?  If so I'd suggest opening a new issue with more reproduction information.

**Comment by JackBekket** [comment]
No, actually, I've just upgrade it to last version and everything starts work fine, thank you!

I didn't notice at first place that I forget to bump my version, sorry ¯\_(ツ)_/¯

**Comment by haltman-at** [comment]
OK, I'm going to leave this closed then, thanks!
